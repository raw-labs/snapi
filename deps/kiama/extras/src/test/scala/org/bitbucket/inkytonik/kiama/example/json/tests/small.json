{
   "one"   : 1.7,
   "two"   : "two",
   "three" : true
}
