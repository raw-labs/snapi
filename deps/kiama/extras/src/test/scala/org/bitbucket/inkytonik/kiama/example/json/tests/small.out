JObject(
 Vector(
  JName("one") -> JNumber(1.7),
  JName("two") -> JString("two"),
  JName("three") -> JTrue()))
{ "one" : 1.7, "two" : "two", "three" : true }
